standalone/src/models/recurring-order/recurring-order/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const recurringOrderState = {
+  Active: 'Active',
+  Paused: 'Paused',
+  Expired: 'Expired',
+  Canceled: 'Canceled',
+} as const;
```

Comment:
I think we should be using the `enum` from the generated types. Otherwise, I think Typescript will throw an error if we try to assign a string to a typed enum value.
```suggestion
import { TCtpRecurringOrderState } from '@/graphql-types';

export const recurringOrderState = TCtpRecurringOrderState;
```